FaunaDB.Client/Types/ObjectV.cs

```diff
@@ -14,13 +14,14 @@ namespace FaunaDB.Types
     public sealed class ObjectV : Value
     {
         #region Construction
-        public static readonly ObjectV Empty = new ObjectV(new OrderedDictionary<string, Value>());
+        public static readonly ObjectV Empty =
+            new ObjectV(ImmutableDictionary<string, Value>.Empty);
 
         public IReadOnlyDictionary<string, Value> Value { get; }
 
-        public ObjectV() : this(Empty.Value) { }
+        internal ObjectV() : this(Empty.Value) { }
 
-        public ObjectV(IReadOnlyDictionary<string, Value> value)
+        internal ObjectV(IReadOnlyDictionary<string, Value> value)
```

Comment:
Ops, `ObjectV` is only created when parsing the response, `Obj()` create unwrapped objects